src/app.js

```diff
@@ -25,6 +29,19 @@ app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
+app.use(
+  '/api',
+  jwt({
+    secret: process.env.SECRET_KEY,
+    algorithms: ['HS256'],
+    getToken: (req) => req.signedCookies.token,
```

Comment:
For the line 37, I think you should add this `getToken: (req) => req.signedCookies.token ?? req.cookies.token,` if there is no signed cookie then there should be a regular cookie